style.css

```diff
@@ -0,0 +1,190 @@
+:root{
+  --f1:  #b1abc3; 
+  --f2:  #5c5470; 
+  --f3:  #352f44; 
+  --f4:  #2a2438; 
+  }
+
+body {
+  font-family: Arial, sans-serif;
+  font-size: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 100%;
```

Comment:
this value should be  `min-height: 100vh;` which applies a value of 100% of the viewport height